src/app/home/navbar/services/navigation.service.ts

```diff
@@ -0,0 +1,17 @@
+import { Injectable } from '@angular/core';
+import { LectureService } from '../../../lecture/services/lecture.service';
+import { ContentNode } from '../navbar.component';
+
+@Injectable({
+  providedIn: 'root'
+})
+export class NavigationService {
```

Comment:
I don't think we need a NavigationService just for this. The LectureService can provide us with lectures that we can map to our nodes.